### ISSUE TITLE
SynchronizedBatchNorm2d

### ISSUE THREAD

**Issue by yuhaoliu7456** [jfzhang95/pytorch-deeplab-xception#159]
I saw you made a change in modeling/deeplab.py ,like this:
if isinstance(m[1], nn.Conv2d) or isinstance(m[1], SynchronizedBatchNorm2d) \
                        or isinstance(m[1], nn.BatchNorm2d):
And I wonder what the difference between this new version and the older.
